### PR TITLE
Add service to expose api

### PR DESCRIPTION
**Pull request by travisperson**
Allows exposing the api over a load balancer

### CODE REVIEW

**Review by ognots — APPROVED** [review]
LGTM!

File: charts/lotus-fullnode/templates/service-load-balancer.yaml [hunk 1]
```diff
@@ -26,3 +26,31 @@ spec:
...
+spec:
+  type: LoadBalancer
+  selector:
+    app: lotus-fullnode-app
+    release: {{ .Release.Name }}
+  ports:
+    - protocol: TCP
+      port: {{ .Values.ports.api }}
+      targetPort: 1234
+      name: pi
```
Comment (coryschwartz) [inline charts/lotus-fullnode/templates/service-load-balancer.yaml:55]:
```suggestion
      name: api
```

File: charts/lotus-fullnode/templates/service-load-balancer.yaml [hunk 2]
```diff
@@ -26,3 +26,31 @@ spec:
...
+    release: {{ .Release.Name }}
+{{- with .Values.additionalLabels }}
+{{ toYaml . | indent 4 }}
+{{- end }}
+  annotations:
+{{- with .Values.apiLoadBalancer.annotations }}
+{{ toYaml . | indent 4 }}
+{{- end }}
+spec:
+  type: LoadBalancer
```
Comment (coryschwartz) [inline charts/lotus-fullnode/templates/service-load-balancer.yaml:47]: Why LoadBalancer and not ClusterIP? Is there a reason to expose this to the outside?

**Comment by placer14** [comment]
Closing stale PR.

**Pull request closed without merging**